src/evoKnowledgeShare/evoKnowledgeShare.Backend/Repositories/NoteRepository.cs

```diff
@@ -0,0 +1,116 @@
+﻿using evoKnowledgeShare.Backend.DataAccess;
+using evoKnowledgeShare.Backend.Models;
+
+namespace evoKnowledgeShare.Backend.Repositories
+{
+    public class NoteRepository : Repository<Note>
+    {
+        public NoteRepository(EvoKnowledgeDbContext dbContext) : base(dbContext)
+        { }
+
+        public override void Add(Note entity)
+        {
+            myDbContext.Notes.Add(entity);
+        }
+
+        public override async Task AddAsync(Note entity)
+        {
+            await myDbContext.Notes.AddAsync(entity);
+        }
+
+        public override void AddRange(IEnumerable<Note> entities)
```

Comment:
Szerintem ez így is jó ahogy írtad viszont mivel a Repository a DbContext-ből származik le ezért miért nem lehet használni az ott lévő AddRange metódust ?

https://learn.microsoft.com/en-us/dotnet/api/system.data.entity.dbset.addrange?view=entity-framework-6.2.0

Reply:
Teljesen igazad van, minnel alacsonyabb szinten "semmire nem tamaszkodva" akartam eloszor megirni aztan majd kiszepitem, csak az elmaradt, igy is van meg mit szepiteni rajta